### PR TITLE
[WIP] Adding Lucene.Net

### PR DESCRIPTION
**Pull request by geobmx540**
Will try to gather project members feedback - hopefully prior to accepting this PR

### CODE REVIEW

**Comment by shiftkey** [comment]
@geobmx540 I've set this PR to `[WIP]` to ensure we don't merge it early. Let me know when it's ready to review.

**Comment by geobmx540** [comment]
Alright, ready to go

File: _data/projects/lucene.net.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+tags:
+- .NET
+- C#
+- full-text
+- fulltext
+- lucene
+- search
+upforgrabs:
+- name: get involved
+- link: https://issues.apache.org/jira/browse/LUCENENET/
```
Comment (shiftkey) [inline _data/projects/lucene.net.yml:13]: Is there a more specific URL you could provide here to help users navigate straight to a list of issues which are good for new contributors?

I was hoping to see a label here that might suit:

screen shot 2015-04-22 at 5 15 04 pm

Comment (geobmx540) [inline _data/projects/lucene.net.yml:13]: We do a terrible job of having anything in JIRA, we mostly stick to our mailing list. We agreed amongst ourselves to fill out JIRA, if you want to move this back to WIT for the moment, OK. I will try to respond in a few days with a better URL and a bunch of issues.

**Comment by shiftkey** [comment]
> OK. I will try to respond in a few days with a better URL and a bunch of issues.

Thanks!

**Comment by shiftkey** [comment]
Closing this out as inactive. Feel free to resubmit a PR if the friendly URL issue gets resolved.

**Pull request closed without merging**